cmd/handler-utils.go

```diff
@@ -97,6 +97,11 @@ func path2BucketAndObject(path string) (bucket, object string) {
 	return bucket, object
 }
 
+var userMetadataKeyPrefixes = []string{
```

Comment:
A comment.. would be necessary for a global variable. If this is not used anywhere define them inside the extractMetadataFromHeader() .